src/lib/command_line_tests/dune

```diff
@@ -0,0 +1,7 @@
+(library
+  (name command_line_tests)
+  (public_name command_line_tests)
+  (inline_tests)
+  (libraries async)
```

Comment:
Or, maybe we can somehow make this depend on `coda.exe` and dune will make sure it is built for us? Is it possible to depend on an executable?

Reply:
I looked through the `dune` docs for that, didn't find anything. I can print a message and fail if the executable's not there.